apps/pipelines/nodes/nodes.py

```diff
@@ -413,6 +413,9 @@ def get_output_map(self):
 
 class RouterMixin(BaseModel):
     keywords: list[str] = Field(default_factory=list, json_schema_extra=UiSchema(widget=Widgets.keywords))
+    tag_output_message: bool = Field(
+        default=False, description="Tag output message with route", json_schema_extra=UiSchema(widget=Widgets.checkbox)
```

Comment:
Maybe elaborate a little bit more? "Tag the output message with chosen route" or something like that